History_Of_Changes

```diff
@@ -0,0 +1,16 @@
+7th March 2019
```

Comment:
If you're going to do this file, I'd prefer it to be:

- Called CHANGELOG.md
- Written in Markdown format.

I'd be very inclined to not even bother with a history of changes, given the nature of Git, but it probably would be nice.

Reply:
The changes are more for my reference more than anything, It was started off git so it was part of the port when everything went together.

Reply:
Ok, fair point. I mean, I can probably fix up myself if it matters.